docs/templates/ADOC.tmpl

```diff
@@ -0,0 +1,19 @@
+[caption=]
+.Environment variables for the frontend extension
```

Comment:
`.Environment variables for the frontend extension`

The name `frontend` needs to be replaced by the name of the extension the file is created for. Another example would be app-provider ect...

Reply:
Adapted in https://github.com/owncloud/ocis/pull/3659/commits/4e0e19dfc7d4a6db4d036c76cf36b70e29736589